src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,13 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        User userDefinedByEmail = userService.findByEmail(email);
+        if (userDefinedByEmail != null) {
+            return userDefinedByEmail.getPassword().equals(password);
+        }
         return false;
```

Comment:
you can simplify your code :

```suggestion
        return userDefinedByEmail != null && userDefinedByEmail.getPassword().equals(password);
```